packages/Panel/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@paprika/panel",
-  "version": "1.0.4-alpha.2",
+  "version": "2.0.0",
```

Comment:
🤔 @allison-c its ok to update the version manually?

Reply:
If you want to update the version manually better to run `yarn lerna publish --force-publish=@paprika/panel`, we shouldn't need to manually update the verison in package.json